### ISSUE TITLE
hide-body gives "Cannot read property 'offsetWidth' of undefined'" error

### ISSUE THREAD

**Issue by kkojot** [antoniandre/vue-cal-v4#312]
Hi,

using `hide-body` gives me an error like this:
```javascript
TypeError: Cannot read property 'offsetWidth' of undefined
    at VueComponent.alignWithScrollbar (vuecal.common.js?7fa7:9673)
    at VueComponent.mounted (vuecal.common.js?7fa7:9749)
    at invokeWithErrorHandling (vue.runtime.esm.js?2b0e:1854)
    at callHook (vue.runtime.esm.js?2b0e:4219)
    at Object.insert (vue.runtime.esm.js?2b0e:3139)
    at invokeInsertHook (vue.runtime.esm.js?2b0e:6346)
    at Vue.patch [as __patch__] (vue.runtime.esm.js?2b0e:6565)
    at Vue._update (vue.runtime.esm.js?2b0e:3945)
    at Vue.updateComponent (vue.runtime.esm.js?2b0e:4060)
    at Watcher.get (vue.runtime.esm.js?2b0e:4479)
```
reproduction link: https://codepen.io/kkojot/pen/dyGJNdP

**Comment by antoniandre** [comment]
Thanks for reporting @kkojot and for the reproduction link.
Will let you know when fixed.

**Comment by antoniandre** [comment]
Hi @kkojot, this is now fixed in version 3.6.2.
Let me know if you are still facing an issue!
